app/src/main/java/org/simple/clinic/scheduleappointment/ScheduleAppointmentUpdate.kt

```diff
@@ -31,11 +31,12 @@ class ScheduleAppointmentUpdate(
       ManuallySelectAppointmentDateClicked -> dispatch(ShowDatePicker(model.selectedAppointmentDate!!.scheduledFor))
       is AppointmentFacilitiesLoaded -> appointmentFacilityLoaded(model, event)
       is PatientFacilityChanged -> next(model.appointmentFacilitySelected(event.facility))
-      is AppointmentDone -> scheduleManualAppointment(model)
+      AppointmentDone -> scheduleManualAppointment(model)
       is AppointmentScheduled -> next(model.doneButtonStateChanged(ButtonState.SAVED), CloseSheet)
       SchedulingSkipped -> dispatch(LoadPatientDefaulterStatus(model.patientUuid))
       is PatientDefaulterStatusLoaded -> scheduleAutomaticAppointment(event, model)
       is TeleconsultRecordLoaded -> teleconsultRecordLoaded(model, event)
+      OpenTeleconsultStatusSheet -> dispatch(GoToTeleconsultStatusSheet(model.teleconsultRecord!!.id))
```

Comment:
When the next button is clicked, we would have to schedule the appointment before opening the teleconsult status sheet.

Reply:
I am sorry, looks like I forgot to add this to the description of the story. Doing it now.